src/mumble/ListenerLocalVolumeDialog.cpp

```diff
@@ -17,12 +18,11 @@ ListenerLocalVolumeDialog::ListenerLocalVolumeDialog(ClientUser *user, Channel *
 	: QDialog(parent), m_user(user), m_channel(channel) {
 	setupUi(this);
 
-	m_initialAdjustemt = Global::get().channelListenerManager->getListenerLocalVolumeAdjustment(m_channel->iId);
+	m_initialAdjustment = Global::get().channelListenerManager->getListenerLocalVolumeAdjustment(m_channel->iId);
 
-	// Decibel formula: +6db = *2
-	// Calculate the db-shift from the set volume-faactor
-	float fdbShift = log2f(m_initialAdjustemt) * 6;
-	qsUserLocalVolume->setValue(static_cast< int >(roundf(fdbShift)));
+	// Calculate the db-shift from the set volume-factor
+	int fdbShift = qRound(VolumeAdjustment::toDBAdjustment(m_initialAdjustment));
```

Comment:
Yes. I chose qRound, because UserLocalVolumeDialog was using it and wanted a consistent way to do that (The advantage was, that qRound will return an int already). But I will gladly replace the qRounds later.

Reply:
Yeah used to be used in older code but
a) Even Qt themselves discourage their usage
b) We are slowly working to reduce our dependence on Qt wherever possible